packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -272,6 +298,15 @@ export class WaferMap extends FoundationElement {
 
     private diesChanged(): void {
         this.waferMapUpdateTracker.track('dies');
+        this.renderStrategy = 'main';
+        this.waferMapUpdateTracker.queueUpdate();
+    }
+
+    private diesTableChanged(): void {
+        this.waferMapUpdateTracker.track('dies');
+        if (this.diesTable !== undefined) {
```

Comment:
Why is the `worker` strategy set conditionally but the `main` strategy set unconditionally? Will the component be in a bad state if it tries to queue an update where someone set `diesTable = undefined` but never set `dies`? Or if they set them both to valid values?

I'm not saying we need to gracefully handle switching back and forth between renderers but I do want to avoid an in-between state where some data is set for each renderer and it's unclear which one should be used.

Reply:
I made both conditional. I think when both are set the latest one will be evaluated when the update queue gets popped. Right now the worker render strategy will try and calculate based on the empty dies api, but that will cahnge in the future updates. If the dies are going to be populated afterwards, the calculations will be redone based on the changes. Is there a better strategy to follow? I think we will adapt this along as we keep developing the new rendering